packages/core/src/components/Skeleton/__stories__/Skeleton.stories.tsx

```diff
@@ -12,41 +12,37 @@ export default {
 };
 
 const skeletonTemplate = (args: SkeletonProps) => {
+  return <Skeleton {...args} />;
+};
+
+export const Overview = {
+  render: skeletonTemplate.bind({}),
+  name: "Overview"
+};
+
+export const ComplexExample = () => {
```

Comment:
Had to extract the overview example to a different story to make the overview story a real playable example, need to verify with designers about the new story